src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -43,30 +42,32 @@
         <!-- List Content -->
         <v-row
           class="people-roles-content"
-          :class="{ 'list-item__subtitle': !isSummary }"
-          v-for="(officer, index) in personList"
+          v-for="(orgPerson, index) in personList"
```

Comment:
I changed the iterator name, since "officer" is actually a field within orgPerson.